uber/model_checks.py

```diff
@@ -294,30 +294,11 @@ def emergency_contact_not_cellphone(attendee):
 
 
 @validation.Attendee
-def printed_badge_deadline(attendee):
```

Comment:
I noticed this while looking through the model checks for any references to PRINTED_BADGE_DEADLINE -- this is actually a duplicate validation! We have another, more robust version of it called `no_more_custom_badges`. It does the same thing and guards against the badge type changing, so I got rid of this one.